WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverFragment.kt

```diff
@@ -58,9 +66,22 @@ class ReaderDiscoverFragment : Fragment(R.layout.reader_discover_fragment_layout
                     is SharePost -> ReaderActivityLauncher.sharePost(context, post)
                     is OpenPost -> ReaderActivityLauncher.openPost(context, post)
                     is ShowReaderComments -> ReaderActivityLauncher.showReaderComments(context, blogId, postId)
+                    ShowNoSitesToReblog -> ReaderActivityLauncher.showNoSiteToReblog(activity)
```

Comment:
```suggestion
                    is ShowNoSitesToReblog -> ReaderActivityLauncher.showNoSiteToReblog(activity)
```

Reply:
I'll need to think about this. The `is` is "missing" here, because `ShowNoSitesToReblog` is an object. On one hand I think it's nice to see it's an object even in the `when`. On the other hand I agree it looks a bit weird that the `is` isn't there.